tests/test_ingest.py

```diff
@@ -0,0 +1,34 @@
+import pathlib
+
+from workflow_ephys.ingest import (ingest_subjects, ingest_sessions)
+from workflow_ephys.pipeline import (subject, lab, ephys, probe, Session,
+                                     get_ephys_root_data_dir)
+
+from . import dj_config, subjects_csv, sessions_csv, kilosort_paramset
+
+
+def test_ingest_subjects(subjects_csv):
```

Comment:
The idea would be to add fixture as a dependency for this test and then verify it. For instance:
```suggestion
def test_ingest_subjects(subjects_csv, subject):
```